package.json

```diff
@@ -17,14 +17,15 @@
     "compile": "tsc ./tsconfig.build.json --resolveJsonModule",
     "lint": "eslint --fix src",
     "style": "prettier --write ./src/**/*.ts",
-    "test": "mocha -r ts-node/register 'test/**/*.test.ts' --exit --config test/.mocharc.json --timeout 120000",
+    "test": "mocha -r ts-node/register 'test/**/*.test.ts' --exit --config test/.mocharc.json --timeout 000",
```

Comment:
whats up with the 000?

Reply:
somehow wasn't deleted with the rest of it